src/main/java/org/sasanlabs/service/vulnerability/fileupload/UnrestrictedFileUpload.java

```diff
@@ -295,6 +301,7 @@ Path getContentDispositionRoot() {
                     "UNRESTRICTED_FILE_UPLOAD_IF_FILE_NAME_NOT_CONTAINS_.PNG_OR_.JPEG_CASE_INSENSITIVE")
     @VulnerableAppRequestMapping(
             value = LevelConstants.LEVEL_6,
+            variant = Variant.SECURE,
```

Comment:
Please take the code update, you will be able to break this level.

Reply:
You are right, I did not notice the meaning of change. Thanks